src/containers/search/SearchForm.tsx

```diff
@@ -65,6 +70,12 @@ export const SearchForm: React.FC = () => {
   const theme = useTheme();
   const { pathname, search } = useLocation();
 
+  // monitor error state locally
+  // generic error message
+  const [showError, setShowError] = useState<unknown>(undefined);
```

Comment:
It is always better to handle errors as strings. Using `unknown` as a type of possible `showError` value can lead to unpredictable issues in future. You will never be sure which kind of object is here.
I propose, in case of an error to extract (or create) a message from the error and initialize the `showError` state with this value.
This way the state definition will be:
```
const [showError, setShowError] = useState<string | undefined>();
```

Reply:
The error that this state is monitoring comes from `useAccommodationsAndOffers` hook. It is an `unknown` type from the source. Ideally, it should be an `Error` object. I am not sure why it is `unknown `.  

Maybe, I should try to get at the source and type it properly?

Reply:
> I am not sure why it is unknown .
obviously, because this property has not been properly typed inside `useAccommodationsAndOffers`.
I if want, you can fix this.

Reply:
Great! I will do that